### PR TITLE
Feature/forgot password page

### PR DESCRIPTION
**Pull request by Ganeshdckap**
- [x] User should be able to enter the email id and password should be sent in email
- [x]  user should enter valid email id
- [x]  From the mail,  user should be redirected to new page wheee he can enter new password and confirm password to reset the password and on success should be directed to login page
- [x]  If the new password and confirm password doesn't match, error will be shown
- [x]  Hide the password and show it in clicking eye icon
 - [ ] All password requirement should be shown while typing password

### CODE REVIEW

**Comment by avinashdckap** [comment]
Screenshot from 2023-11-15 12-59-59
app is not loading, dont mention ur system path in the import of images

File: index.html [hunk 1]
```diff
@@ -4,7 +4,8 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <title>Palli Application</title>
```
Comment (avinashdckap) [inline index.html:8]: refer @Parasuramandckap

File: package.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@ant-design/icons": "^5.2.6",
```
Comment (avinashdckap) [inline package.json:13]: refer @Parasuramandckap

Comment (avinashdckap) [reply package.json:13]: uninstalll this package

File: src/App.jsx [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
+import Forgotpassword from "./pages/ForgotScreen";
+import ResetPassword from "./pages/ResetPasswordScreen";
 
 //Define your routes for APP here
 
 const App = () => {
   return (
     <Routes>
       <Route path="/" element={<LoginScreen />}></Route>
+      <Route path="/forgot-password" element={<Forgotpassword />}></Route>
```
Comment (avinashdckap) [inline src/App.jsx:12]: forgot/password
change/password

File: src/assets/css/forgotPage.css [hunk 4]
```diff
@@ -0,0 +1,147 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
```
Comment (avinashdckap) [inline src/assets/css/forgotPage.css:2]: forgotPage.css => ForgotPage.css

move this css files to the resp folder refer @Parasuramandckap

File: src/assets/css/resetPage.css [hunk 5]
```diff
@@ -0,0 +1,110 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+/* Resetting default styles for all elements */
```
Comment (avinashdckap) [inline src/assets/css/resetPage.css:3]: same as above

File: src/assets/images/Forgot-password.png
Comment (avinashdckap) [inline src/assets/images/Forgot-password.png:1]: refer @Parasuramandckap

File: src/pages/ForgotScreen.jsx [hunk 6]
```diff
@@ -0,0 +1,61 @@
+// Import necessary dependencies
+import React from "react";
+import ForgotPasswordImage from "../assets/images/Forgot-password.png";
+import "/home/dckap/Palli-Application/Palli-Application/src/assets/css/forgotPage.css";
```
Comment (avinashdckap) [inline src/pages/ForgotScreen.jsx:4]: dont use path like this

File: src/pages/ForgotScreen.jsx [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+// Define a functional component for the Forgot Password page
+const Forgotpassword = () => {
+  return (
+    <div className="forgot-container">
+      {/* Left side container with an image */}
+      <div className="left-side">
+        <img src={ForgotPasswordImage} alt="ForgotPasswordImage" />
```
Comment (avinashdckap) [inline src/pages/ForgotScreen.jsx:12]: ForgotPasswordImage -> mention the relative path here -> src/assets/images/xxx.png

File: src/pages/ResetPasswordScreen.jsx [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+import Logo from "/home/dckap/Palli-Application/Palli-Application/src/assets/images/logo.png";
+import ResetPasswordImage from "/home/dckap/Palli-Application/Palli-Application/src/assets/images/RestPageImage.png";
+import "/home/dckap/Palli-Application/Palli-Application/src/assets/css/resetPage.css";
+
+const ResetPassword = () => {
+  const [password, setPassword] = useState("");
+  const [confirmPassword, setConfirmPassword] = useState("");
+  const [passwordError, setPasswordError] = useState("");
+
+  const handlePasswordChange = (e) => {
```
Comment (avinashdckap) [inline src/pages/ResetPasswordScreen.jsx:13]: use single handlechange function to handle both changes using e.target.name

File: src/pages/ResetPasswordScreen.jsx [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+                <Input.Password
+                  placeholder="Input password"
+                  value={password}
+                  onChange={handlePasswordChange}
+                />
+                <label htmlFor="ConfirmPassword">
+                  Confirm New Password
+                  <span className="required-symbole">*</span>
+                </label>
+                <Input.Password
```
Comment (avinashdckap) [inline src/pages/ResetPasswordScreen.jsx:65]: Destructure password refer @Parasuramandckap

**Review by avinashdckap — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Ganeshdckap) [reply index.html:7]: I have removed the unnecessary code and removed the path.

File: package.json (hunk 2 above)
Comment (Ganeshdckap) [reply package.json]: I have uninstalled ant design and written normal input code

File: src/App.jsx (hunk 3 above)
Comment (Ganeshdckap) [reply src/App.jsx]: I have changed the router name as you said

forgot/password
change/password

File: src/pages/ForgotScreen.jsx (hunk 6 above)
Comment (Ganeshdckap) [reply src/pages/ForgotScreen.jsx]: I removed the path and changed it.

**Comment by avinashdckap** [comment]
More issue will be added after merging this PR

**Pull request merged**